containers/Loans/Loans.tsx

```diff
@@ -91,11 +107,13 @@ function Container() {
 		const loadLoans = async () => {
 			setIsLoadingLoans(true);
 			const openLoans = await getOpenLoans({
-				loanContracts,
-				loanStateContracts,
+				erc20LoanContract,
+				ethLoanContract,
+				erc20LoanStateContract,
+				ethLoanStateContract,
 				address,
-				network,
 				isL2,
+				subgraphOpenLoans: subgraphOpenLoansKey ? JSON.parse(subgraphOpenLoansKey) : [],
```

Comment:
here we are parsing the subgraphOpenLoansKey. we could simply store it as array instead of stringify